qanta/buzzer/progress.py

```diff
@@ -0,0 +1,74 @@
+import datetime
```

Comment:
I wrote this progressbar to work specifically with my iterator. I wish to keep it for now and probably remove it all together later.